tools/accuracy_checker/accuracy_checker/metrics/classification.py

```diff
@@ -353,3 +350,35 @@ def evaluate(self, annotations, predictions):
     def reset(self):
         self.targets = []
         self.results = []
+
+class AcerScore(PerImageEvaluationMetric):
+    __provider__ = 'acer_score'
+    annotation_types = (ClassificationAnnotation, TextClassificationAnnotation)
+    prediction_types = (ClassificationPrediction, )
+
+    def configure(self):
+        if isinstance(confusion_matrix, UnsupportedPackage):
+            confusion_matrix.raise_error(self.__provider__)
+        self.reset()
+
+    def update(self, annotation, prediction):
+        self.targets.append(annotation.label)
+        self.results.append(prediction.label)
+        return (prediction.label == annotation.label)
+
+    def evaluate(self, annotations, predictions):
+        all_results = np.array(self.results)
+        all_targets = np.array(self.targets)
+        tn, fp, fn, tp = confusion_matrix(y_true=all_targets,
+                                          y_pred=all_results,
+                                          ).ravel()
+
+        apcer = fp / (tn + fp) if fp != 0 else 0
+        bpcer = fn / (fn + tp) if fn != 0 else 0
```

Comment:
Why did you use `fp != 0` condition here? May be `fp + tn != 0` more suitable here?